tests/integrations/realcluster/deploy.sh

```diff
@@ -15,9 +15,15 @@ curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh
 $TIUP_BIN_DIR update playground
 
 cd ../../..
-# Run TiUP
-$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor \
-	--pd.binpath ./bin/pd-server --kv.binpath ./bin/tikv-server --db.binpath ./bin/tidb-server --tiflash.binpath ./bin/tiflash --tag pd_test \
-	> $CUR_PATH/playground.log 2>&1 &
+if [ ! -d "bin" ] || [ ! -e "bin/tikv-server" ] && [ ! -e "bin/tidb-server" ] && [ ! -e "bin/pd-server" ] && [ ! -e "bin/tiflash" ]; then
+	color-green "check binaries..."
+	$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor --tag pd_test \
+		> $CUR_PATH/playground.log 2>&1 &
+else
+  color-green "using existing binaries..."
+	$TIUP_BIN_DIR playground nightly --kv 3 --tiflash 1 --db 1 --pd 3 --without-monitor \
+		--pd.binpath ./bin/pd-server --kv.binpath ./bin/tikv-server --db.binpath ./bin/tidb-server --tiflash.binpath ./bin/tiflash --tag pd_test \
```

Comment:
Does the user need to move bin to the current bin path?

Reply:
Currently, we download the binaries via the `ci` repo https://github.com/PingCAP-QE/ci/blob/main/pipelines/tikv/pd/latest/pull_integration_realcluster_test.groovy#L57- L65 in `pd/bin` to download the binary.

This approach reduces the cost of learning for developers. However, these packages can only be downloaded by the ci, which means that it is not possible to run locally.

To solve this problem, I would like to load the packages via tiup instead of downloading them by self.

- Because for example, tiflash is a bit tricky to download packages
https://github.com/PingCAP-QE/ci/blob/main/scripts/artifacts/download_pingcap_artifact.sh#L101
- maybe I can give more information in annotation, or do you have any advice? :)

Reply:
I see. Maybe we provider some flags to specify bin paths for other components?

Reply:
I'm not sure we need to support this feature
I think both methods start the test in essentially the same way?
- cp binary file to directory
- specific binpath flags

cc @JmPotato

Reply:
Yes, It's both OK to me.

Reply:
I think we could keep the current implementation since it's most used in the CI environment.